# <center/>富集分析

$$\begin{cases}
GO与KECG
\begin{cases}
GO功能注释与KECG通路\\
ORA(过表达分析)
\end{cases}\\
GESA
\begin{cases}
GESA的原理\\
MsigDB的数据类型\\
GESA的软件与操作步骤
\end{cases}
\end{cases}
$$

## 1:GO分析与KECG分析

### &emsp;GO功能注释(Gene Oncology)与 KEGG通路
&emsp;GO数据库,全称是Gene Ontology(基因本体),基因的功能被分为三个部分,分别为:**细胞组分**(cellular component,CC),**分子功能**(molecular function,MF),**生物过程**(biological process,BP)。利用GO数据库,可以得到目标基因在CC,MF,BP三个层面上与什么有关。
&emsp;KECG是介绍基因参与人体相关通路的数据库。

### &emsp;ORA(Over Representation)过表达分析
&emsp;对基因差异表达的程度设定阈值,差异程度达到阈值的基因可作为候选分析的基因。作图:横坐标为差异表达的logP值,纵坐标为该基因的相关通路或功能。

## 2:基因组富集分析(GESA)

### &emsp;GESA的原理(Gene Set Enrichment Analysis, GSEA)
&emsp;基因集富集分析 (Gene Set Enrichment Analysis, GSEA) 的基本思想是使用预定义的基因集（通常来自功能注释或先前实验的结果），将**基因集按照在两类样本中的差异表达程度排序**，然后**检验预先设定的基因集合**是否在这个排序表的顶端或者底端**富集**。差异量化方法有Signal2Noise值、Ttest值、或者是fold change值、logFC值，默认为signal-to-noise metric。GESA使用MSigDB作为背景数据集。

>**GESA相较于GO，KEGG的优势**
>一般的差异分析（GO和Pathway）往往侧重于比较两组间的基因表达差异，**集中关注少数几个显著上调或下调的基因**，这容易遗漏部分差异表达不显著却有重要生物学意义的基因，忽略一些基因的生物特性、基因调控网络之间的关系及基因功能和意义等有价值的信息。因为实际通过芯片观测到的RNA 表达变化，往往是层层的负反馈调控后的结果，并且不同组织对于表达差异的敏感度是不同的：在神经递质系统内，<u>一个1.2倍的表达差异即可能产生及其显著的效应</u>。
而GSEA不需要指定明确的差异基因阈值，算法会根据实际数据的整体趋势， 为研究者们提供了一种合理地解决目前芯片分析瓶颈问题的方法,即使在没有先验经验存在的情况下也能在表达谱整体层次上对数条基因进行分析,从而从数理统计上把表达谱芯片数据与生物学意义很好地衔接起来,使得研究者们能够更轻松、更合理地解读芯片结果。

### &emsp;GESA的步骤

MSigDB数据库(Molecular Signatures Database)定义了已知的基因集合，包括H和C1-7八个系列(Collection)
|abbr|definition|定义|内容|
|---|---|---|---|
H|hallmark gene sets|Hallmark gene sets summarize and represent specific well-defined biological states or processes and display coherent expression. These gene sets were generated by a computational methodology based on identifying overlaps between gene sets in other MSigDB collections and retaining genes that display coordinate expression.|共50个，如细胞凋亡特征基因集
C1|positional gene sets|Gene sets corresponding to each human chromosome and each cytogenetic band|根据染色体位置，共326个
C2|curated gene sets|Gene sets in this collection are cu+rated from various sources, including online pathway databases and the biomedical literature. Many sets are also contributed by individual domain experts. The gene set page for each gene set lists its source.|CGP(Chemical and genetic perturbation)<br/> CP(Canonical Pathway)<br/>  CP BIOCARTA(BioCarta gene sets)<br/> CP KECG(KECG gene sets)<br/> CP REACTOME(Reactome gene sets)<br/>
C3| regulatory target gene sets|Gene sets representing potential targets of regulation by transcription factors or microRNAs. The sets consist of genes grouped by elements they share in their non-protein coding regions. The elements represent known or likely cis-regulatory elements in promoters and 3'-UTRs.|MIR(microRNA targets)<br/> TFT(transciption factor targets)
C4|computational gene sets|Computational gene sets defined by mining large collections of cancer-oriented microarray data. |通过挖掘癌症相关芯片数据定义的基因集合<br/> CGN(cancer gene neighbour)<br/> CM(cancer modules)
C5|GO gene sets|Gene sets that contain genes annotated by the same ontology term.|BP(GO biological process)<br/> CC(GO cellular component)<br/> MF(GO molecular function)<br/> HPO(Human Phenotype Ontology)
C6|oncogenic signatures|Gene sets that represent signatures of cellular pathways which are often dis-regulated in cancer. The majority of signatures were generated directly from microarray data from NCBI GEO or from internal unpublished profiling experiments involving perturbation of known cancer genes.|大部分来源于NCBI GEO 未发表芯片数据
C7|Immunologic signature|Gene sets that represent cell states and perturbations within the immune system. The signatures were generated by manual curation of published studies in human and mouse immunology. 
C8|cell type signature gene set|Gene sets that contain curated cluster markers for cell types identified in single-cell sequencing studies of human tissue. 

**GESA软件下载地址 http://software.broadinstitute.org/gsea/downloads.jsp**

#### 1 计算富集分数

#### 2 估计富集分数的显著性水平

#### 3 矫正多重假设检验



## 参考资料
1 GESA使用手册 https://www.gsea-msigdb.org/gsea/doc/GSEAUserGuideFrame.html
2 http://software.broadinstitute.org/gsea/msigdb
 微信 小张聊科研 GESA分析是个什么鬼